### PR TITLE
Feature/homepage cfp

### PR DESCRIPTION
**Pull request by joren**

### CODE REVIEW

File: _source/_includes/index_content.html [hunk 1]
```diff
@@ -5,8 +5,10 @@
       Our speakers present a wide range of talks, from Ruby or Rails on the technical side, to inspirational talks about managing teams or communicating, all the way to front-end development and design. We’ve even been taught how to survive family parties!
     </p>
     <p>
-      The Call for Proposals will be announced shortly. In the mean time you can
-      find out more about the process <a href="/cfp/">here</a>.
+      The Call for Proposals is open, submit your talk and join our ship.
```
Comment (toch) [inline _source/_includes/index_content.html]: Add a comma before `and join`

File: _source/_includes/navigation.html [hunk 2]
```diff
@@ -2,6 +2,8 @@
   <ul>
     <!-- <li class="{% if page.class == 'speakers' %}active{% endif %}"><a href="/speakers">Speakers</a></li> -->
     <!-- <li class="{% if page.class == 'schedule' %}active{% endif %}"><a href="/schedule">Schedule</a></li> -->
+    <li class="{% if page.class == 'cpf' %}active{% endif %}"><a
```
Comment (toch) [inline _source/_includes/navigation.html]: shouldn't be `cfp` rather than `cpf`?

Comment (toch) [inline _source/_includes/navigation.html]: yes it should be `cfp`, check provided a new token, record a new cassette, change the code to match `pullreviewtester` user

**Comment by joren** [comment]
Fixed

**Pull request merged**